canvas_modules/harness/package.json

```diff
@@ -38,6 +38,7 @@
     "@carbon/charts-react": "1.15.3",
     "@carbon/react": "^1.52.0",
     "@elyra/canvas": "file:../common-canvas",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.11",
```

Comment:
Remove `react-hot-loader` from package.json and pin this dependency as well.

Reply:
Hi @nmgokhale I have removed react-hot-loader package and pinned react-refresh-webpack-plugin in package.json, please review.